### PR TITLE
Improved docs

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: docs/README.building [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+* OpenH264
+* FFMPEG (x264 or OpenH264)
+
+There are some platform specific implementations too (e.g. mediacodec on android) but these
+two are the options that are always required.
+
+5. Graphics scaling support
+
+High DPI modes and the client side smart-sizing option require bitmaps to be scaled.
```
Comment (pnowack) [inline docs/README.building:46]: Let's add one more item here: "and the ability to let the server side to scale content". The 105, 106, and 107 GFX caps require the client side to be able to scale bitmaps.

File: docs/README.building [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+Azure logon requires HTTP/JSON messages to be parsed.
+
+We support:
+
+* cJSON
+
+4. H264
+
+RDP GFX modes (anything newer Windows 8.1 / Server 2008) supports a graphics mode based
```
[the remark this answers is not in the document]
Comment (akallabeth) [reply docs/README.building]: good catch

File: docs/README.building (hunk 1 above)
Comment (akallabeth) [reply docs/README.building]: rephrased.

File: docs/README.building [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+We support
+
+* OpenSSL our main development SSL library
+* LibreSSL (supported by community)
+* MBedTLS (supported by community)
+
+2. Kerberos
+
+Authentication to many services requires kerbers (especially if smartcards are in use)
```
Comment (mfleisz) [inline docs/README.building:16]: Missing `o` in `kerbers`

File: docs/README.building [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+High DPI support and smart-sizing option require bitmaps to be scaled by the client. 
+
+We support
+
+* Swscale
+* Cairo
+
+6. Audio encoders/decoders
+
+Sound and Microphone options allow transmission encoded in compressed formats. While also
```
Comment (mfleisz) [inline docs/README.building:55]: "allow transmission encoded in compressed formats" - looks like there is something missing here i.e. "allow transmission of audio samples encoded in compressed formats".

I'd also remove the `While also` from the next sentence.

File: docs/README.building [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+We support
+
+* PCSC
+* pkcs11
+
+PCSC is required for smartcard redirection, pkcs11 for NLA smartcard logon support
+
+8. Unicode
+
+Most of the protocol preferrably uses UCS-2/UTF16 for strings. To convert to/from UTF-8 a
```
Comment (mfleisz) [inline docs/README.building:77]: preferably (one `r` too much)

File: docs/README.building (hunk 3 above)
Comment (akallabeth) [reply docs/README.building]: nice catch

File: docs/README.building (hunk 5 above)
Comment (akallabeth) [reply docs/README.building]: nice catch

File: docs/README.building (hunk 4 above)
Comment (akallabeth) [reply docs/README.building]: rephrased

**Review by mfleisz — APPROVED** [review]

**Pull request merged**